devinette.py

```diff
@@ -0,0 +1,29 @@
+# jeu de devinette
+import random
+
+n = random.randint(0,100)
+
+appreciation = "?"
+
+while True:
+    var = input("Enter a number :")
+
+    var = int(var)
```

Comment:
this line can be merge with the previous one

Reply:
ofcourse even inside every condition's loop he can merge any two statements .